src/main/java/com/dreamsportslabs/guardian/constant/Constants.java

```diff
@@ -108,4 +109,8 @@ public final class Constants {
   public static final String MESSAGE_TEMPLATE_NAME = "templateName";
   public static final String MESSAGE_TEMPLATE_PARAMS = "templateParams";
   public static final String MESSAGE_TEMPLATE_PARAMS_OTP = "otp";
+
+  public static final String FORMAT_PEM = "PEM";
+  public static final String FORMAT_JWKS = "JWKS";
+  public static final Set<Integer> VALID_KEY_SIZES = Set.of(2048, 3072, 4096);
```

Comment:
Although Set.of makes an Immutable set, java.util.Set in general isn't.
Should ImmutableList be used here?

Open to both, just that Set.of does not suggest immutability on reading, although it is.